lib/Swagger/src/Discussion/Api/SitesApi.php

```diff
@@ -334,4 +304,215 @@ public function getSiteWithHttpInfo($site_id)
             throw $e;
         }
     }
+
+    /**
+     * Operation hardDeleteSite
+     *
+     * Hard delete
+     *
+     * @param int $site_id The id of the site (required)
+     * @param string $x_wikia_schwartz  (optional)
+     * @return \Swagger\Client\Discussion\Models\SiteHalResponse
+     * @throws \Swagger\Client\ApiException on non-2xx response
+     */
+    public function hardDeleteSite($site_id, $x_wikia_schwartz = null)
+    {
+        list($response) = $this->hardDeleteSiteWithHttpInfo($site_id, $x_wikia_schwartz);
```

Comment:
![MINOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-minor.png) Immediately return this expression instead of assigning it to the temporary variable "$response". [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://sonarqube.wikia-inc.com/coding_rules#rule_key=php%3AS1488)